godtools/Managers/LanguagesManager.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  LanguagesManager.swift
+//  godtools
+//
+//  Created by Ryan Carlson on 4/18/17.
+//  Copyright © 2017 Cru. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import Alamofire
+import PromiseKit
+import Spine
+import MagicalRecord
+
+class LanguagesManager: NSObject {
+    static let shared = LanguagesManager()
+    
+    let path = "/languages"
+    let serializer = Serializer()
+    
+    var languages = [Language]()
+    
+    override init() {
+        super.init()
+        serializer.registerResource(LanguageResource.self)
+    }
+    
+    func loadFromDisk() -> Promise<[Language]> {
+        languages = Language.mr_findAll() as! [Language]
```

Comment:
I like all of this! really easy to follow, I believe we can make an "abstract" class, so a lot of these functionalities can be set there, and it should only call and pass the arguments for process the data (when retrieving with MagicalRecord, and when calling to the server with AlamoFire). That would help when making the other managers, it is going to be easier and faster, what do you think?